fe/fe-core/src/main/java/com/starrocks/catalog/ListPartitionInfo.java

```diff
@@ -102,6 +137,19 @@ public static PartitionInfo read(DataInput in) throws IOException {
         list.idToTabletType.forEach((k, v) -> partitionInfo.setTabletType(k, v));
         partitionInfo.setIsMultiColumnPartition();
         partitionInfo.type = list.getType();
+
+        try {
+            Map<Long, List<String>> idToValuesMap = partitionInfo.getIdToValues();
+            for (Map.Entry<Long, List<String>> entry : idToValuesMap.entrySet()) {
+                partitionInfo.setLiteralExprValues(entry.getKey(), entry.getValue());
+            }
+            Map<Long, List<List<String>>> idToMultiValuesMap = partitionInfo.getIdToMultiValues();
+            for (Map.Entry<Long, List<List<String>>> entry : idToMultiValuesMap.entrySet()) {
+                partitionInfo.setMultiLiteralExprValues(entry.getKey(), entry.getValue());
+            }
+        } catch (AnalysisException e) {
+            throw new IOException(e);
```

Comment:
Do not throw exception here. Just print error log. Exception in replay log will cause process exit.